app/helpers/react_components/mentoring/representations/with_feedback.rb

```diff
@@ -24,8 +24,7 @@ def representations_request
             endpoint: Exercism::Routes.with_feedback_api_mentoring_representations_url,
             query: representations_request_params,
             options: {
-              initial_data: representations,
-              stale_time: 5000 # milliseconds
```

Comment:
It is. However, I've relocated them to the frontend. Given that the fetching occurs on the frontend, it seems logical to manage the corresponding options there rather than on the backend.